main.rb

```diff
@@ -0,0 +1,12 @@
+require './users.rb'
+require './database_setup.rb'
+
+
+# setup_database()
+
+
+
+u1=User.new('chinku','chinku@gmail.com','password','user')
```

Comment:
space around `=`

Reply:
I'd recommend key-value arguments here so that there is no doubt when initializing. currently all 4 arguments are string args and one can pass anything